airbyte-integrations/connectors/destination-mysql/src/test-integration/java/io/airbyte/integrations/destination/mysql/MySQLDestinationAcceptanceTest.java

```diff
@@ -252,6 +252,46 @@ public void testLineBreakCharacters() {
     // overrides test with a no-op until we handle full UTF-8 in the destination
   }
 
+    private String retrieveTableCharset(final String schemaName, final String tableName) throws SQLException {
+        try (final DSLContext dslContext = DSLContextFactory.create(
+                db.getUsername(),
+                db.getPassword(),
+                db.getDriverClassName(),
+                String.format(DatabaseDriver.MYSQL.getUrlFormatString(),
+                        db.getHost(),
+                        db.getFirstMappedPort(),
+                        db.getDatabaseName()),
+                SQLDialect.MYSQL)) {
+            return new Database(dslContext).query(
+                    ctx -> ctx
+                            .fetch(String.format("SELECT CCSA.character_set_name \n" +
+                                    "FROM information_schema.`TABLES` T,information_schema.`COLLATION_CHARACTER_SET_APPLICABILITY` CCSA\n" +
+                                    "WHERE CCSA.collation_name = T.table_collation\n" +
+                                    "AND T.table_schema = \"%s\" AND T.table_name = \"%s\";", schemaName, tableName))
+                            .stream().map(this::getJsonFromRecord).toList().get(0).get("CHARACTER_SET_NAME").asText());
+        }
+    }
+
+    @Test
+    public void testUTF8() throws Exception {
```

Comment:
How are we testing the `LOAD DATA INFILE` query in `MySQLSqlOperations` class via this test? I would want to have a test where we try to load data into a table via the query present in `MySQLSqlOperations`

Reply:
I have added test about `LOAD DATA INFILE` query